bin/crew

```diff
@@ -1069,7 +1077,7 @@ end
 def remove (pkgName)
 
   #make sure the package is actually installed
-  unless @device[:installed_packages].any? { |pkg| pkg[:name] == pkgName }
+  unless @device[:installed_packages].any? { |pkg| pkg[:name] == pkgName } || File.file?("#{CREW_META_PATH}#{pkgName}.filelist")
```

Comment:
Should this be `File.exist?` instead of `File.file?`?

Reply:
`File.file?` is used elsewhere, which is why I used it there.  `File.exist?` would work fine too. 

If it is worth it I can change it...

Reply:
Doesn't `File.file?` just determine if it's a file and then fallback to false if it doesn't find it?  `File.exist?` returns a boolean true or false based on the true intention of what we are trying to determine.

Reply:
Changed.